source/_integrations/androidtv_remote.markdown

```diff
@@ -0,0 +1,382 @@
+---
+title: Android TV Remote
+description: Instructions on how to integrate Android TV remotes into Home Assistant.
+ha_category:
+  - Remote
+ha_release: 2023.4
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@tronikos'
+ha_quality_scale: platinum
+ha_domain: androidtv_remote
+ha_zeroconf: true
+ha_platforms:
+  - diagnostics
+  - remote
+ha_integration_type: device
+---
+
+The `androidtv_remote` platform allows you to control an Android TV device by sending [commands](https://github.com/tronikos/androidtvremote2/blob/main/TvKeys.txt) and launching apps.
+
+{% include integrations/config_flow.md %}
+
+## Entity
+
+This integration adds a `remote` entity which turns on/off the Android TV device.
+The entity has the `current_activity` attribute that shows the current foreground app on the Android TV.
+
+## Services
+
+### Service `remote.turn_off`
```

Comment:
This isn't a service provided by this integration, but by the `remote` integration.
Let's remove it here, as we do not need to create duplicate documentation.

Reply:
I followed the documentation for other remote integrations which mention these services
Examples:
https://www.home-assistant.io/integrations/harmony/
https://www.home-assistant.io/integrations/apple_tv/#remote
https://www.home-assistant.io/integrations/xbox/#remote

Reply:
Maybe remove the attributes and the documentation for the services and just have examples for each service? All the other integrations I linked to in my previous comment list the service attributes though. I see some value in that since not all integrations support all the attributes. e.g. I think only one other integration supports the hold_secs attribute of send_command. My integration supports all the attributes of send_command but then it doesn't support learn_command. So I think there is value explicitly listing all the supported services and how to call them.
I also just noticed the documentation at https://www.home-assistant.io/integrations/remote/ doesn't have good documentation for the services. Maybe that's intended since not all remote integrations support them and it's better to have each implementation list the supported ones.

Reply:
In addition there is value explaining what each attribute is since it might be slightly different between remote implementations. e.g. Harmony for activity uses presets e.g. Watch TV that can send a series of signals to different devices. In my integration that's app to open, e.g. YouTube.

Reply:
We should not duplicate documentation. It is problematic to keep in sync.

> I also just noticed the documentation at [home-assistant.io/integrations/remote](https://www.home-assistant.io/integrations/remote/) doesn't have good documentation for the services.

Feel free to expand that page in a separate PR.

Reply:
I kept the sections of the services, removed the tables and kept the examples as I think they are useful.

Reply:
I disagree, most certainly with a simple turn on as this one. If you want to provide additional examples with specific use cases, I think that is fine to create (as general examples).

But we should not re-document services from other integrations.

Reply:
I removed the section headers, the 1 sentence explanation of each service, and the simpler turn off and on examples.